CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## [2.2.1] - 2020-12-10
+## [2.2.2] - 2022-03
```

Comment:
nitpick: we can add day for consistency reasons.

Reply:
left it for the release commit, when we'll know the actual day of release